From e9e90ea8afcd2b24a15dcc49a6c2f855a4c25ef8 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Thu, 18 Jan 2024 12:08:31 +0100
Subject: [PATCH 0565/1695] plat-stm32mp1: conf: enable TA PKCS11

Default enable CFG_PKCS11_TA for STM32MP1.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 3114ea87f51350b053f6d20508df55bddea7ff0b plat-stm32mp1: conf: enable TA PKCS11]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: Idc7f2f4219e8845bb701e6b7fe95d8449bba9036
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/352419
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 9aebcd78b..58746db78 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -375,6 +375,9 @@ endif
 CFG_DRIVERS_NVMEM ?= y
 CFG_STM32_TAMP_NVRAM ?= y
 
+# Default enable PKCS11 TA to have PKCS11 tests built in OP-TEE Test
+CFG_PKCS11_TA ?= y
+
 # Default use stm32mp1 PM mailbox context version 3
 CFG_STM32MP1_PM_CONTEXT_VERSION ?= 3
 
-- 
2.47.2

